server/app/services/languageProcessor/TaskMap.java

```diff
@@ -67,31 +71,137 @@ public JsonNode getTicketDescription(String issueKey, JsonNode responseBody) {
    * @param issueKey
    * @return
    */
-  public JsonNode getTicketAssignee(String issueKey, JsonNode responseBody) {
-    if (Extractor.extractString(responseBody, "assignee").equals("[\"Issue Does Not Exist\"]")) {
-      return parseToJson("fail", configuration.getString("error-message.issue-not-found"));
+  public JsonNode IssueAssignee(String issueKey, JsonNode responseBody) {
+    if (Extractor.getIssueAssignee(responseBody, "assignee").equals("[\"Issue Does Not Exist\"]")) {
+      return parseToJson("fail", "Cannot find issue");
     } else {
-      String answer = Extractor.extractString(responseBody, "assignee") + " is working on " + issueKey + ".";
+      String answer = Extractor.getIssueAssignee(responseBody, "assignee") + " is working on " + issueKey + ".";
       System.out.println(answer);
       return parseToJson("success", answer);
     }
   }
 
   /**
-   * This method takes String as an input as returns a JSON object in the required format
+   * This method requests brief description of issue and returns it to the calling method
    *
+   * @param issueKey
+   * @return
+   */
+  public JsonNode IssueBrief(String issueKey, JsonNode responseBody) {
+    if (Extractor.getIssueBrief(responseBody, "assignee").equals("[\"Issue Does Not Exist\"]")) {
+      return parseToJson("fail", "Cannot find issue");
+    } else {
+      String answer = Extractor.getIssueBrief(responseBody, "assignee") + " is working on " + issueKey + ".";
+      System.out.println(answer);
+      return parseToJson("success", answer);
+    }
+  }
+
+  /** COMPLETE THIS METHOD
+   * This method requests status of an issue and returns it to the calling method
+   *
+   * @param issueKey
+   * @return
+   */
+  public JsonNode IssueStatus(String issueKey, JsonNode responseBody) {
+
+    String answer = "Code to find the STATUS OF AN ISSUE has not been implemented yet. Please wait for our next version update.";
+    System.out.println(answer);
+    return parseToJson("success", answer);
+  }
+
+  /**COMPLETE THIS METHOD
```

Comment:
@haolinj I had removed all of these. I don't know how they have appeared again. I'll remove it again.